content/2015.11.shweta-oak.rst

```diff
@@ -0,0 +1,80 @@
+Discovery of Kinto servers
+###########################
+
+:slug: outreachy-2016
+:date: 2015-11-27
+:lang: en
+:url: shweta-oak-kinto-2016
+:summary:
+    Work on Kinto project - making instances discoverable, during internship.
+
+.. epigraph::
+	
+  By having a program targeted specifically towards women, we found that we
+  reached talented and passionate participants, who were uncertain about how to
+  start otherwise.
+
+  We are now expanding the program to people from more groups underrepresented
+  in FOSS. We hope this effort will help many people learn how exciting, varied
+  and valuable work on FOSS projects can be and how inclusive the community
+  really is.
+
+  -- `The GNOME Outreachy program wiki <https://wiki.gnome.org/Outreachy>`_
+
+While looking for projects to contribute to, I read about Kinto, and found it intriguing.
+
+The differences I observed between Kinto and such existing solutions are:
+
+* ­Decentralized discovery of Kinto instances.
+* It is open source and you can self host a kinto server which is not possible in other existing solutions.
+* Fine grained permissions can be imposed on records, collections and buckets which follow a hierarchy.
+* Kinto provides for Synchronization. Applications can synchronize data on different devices.
+* Multiple clients can access a server in case of a shared resource.
+* If an application makes some change, while offline, these changes will be reflected when connection is established.
+
+Decentralization and Discovery 
+===============================
+
+The Kinto project aims to bring storage instances to everyone, attached to their Firefox Accounts. Currently, Kinto is thought of as a centralized server and everyone authenticates on one instance. Items are shared between users of a same instance. Scalability is harder when there is one endpoint, and it's also not easy to exchange and make use of information.
```

Comment:
I don't understand what you want to say when stating "Scalability is harder when there is one endpoint"?